### PR TITLE
Adds five new IPC antennae

### PR DESCRIPTION
**Pull request by Recoherent**
## About The Pull Request
Adds five new antennae options for IPCs to supplement the existing ones. There's only three or four antennae options that don't look like garbage at the moment, so this just gives more choices.
This also removes the "Lights" antenna option, because it was not even real. It was just... in the list. Why?

## Why It's Good For The Game
it's more customization options what more do you want. technically the first thing i've sprited so feedback appreciated etc.
nyaru in real???
horn previews

## Changelog
:cl:
add: IPCs now have five new antennae options! You may recognize one of them from a certain marketable character.
fix: The "lights" option that people kept insisting was a real antenna has been properly scrubbed from reality.
/:cl:

### CODE REVIEW

**Comment by PiperDoots** [comment]
The way you've organised them in the screenshot is a bit strange but great work overall!

**Comment by Vexylius** [comment]
Feeling like this is going to be merged fast

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -2412,6 +2408,26 @@
 	name = "TV Antenna"
 	icon_state = "tvantennae"
 
+/datum/sprite_accessory/ipc_antennas/cross
+	name = "Cross"
+	icon_state = "cross"
+
+/datum/sprite_accessory/ipc_antennas/nyaru
+	name = "Nyaru"
```
Comment (Fikou) [inline code/modules/mob/dead/new_player/sprite_accessories.dm]: rename this to something normal

**Review by triplezeta — APPROVED** [review]
these are some of the antennae of all time

**Pull request merged**